### PR TITLE
fix: update component name in navigator template

### LINKED ISSUE #1854: Navigator Naming Convention
What's going on?

When generating a navigator (e.g. "test") the generated file will be named test-navigator.tsx.
However, the generated component's name is Test and not TestNavigator, which would be consistent with the AppNavigator component. 

 I wonder why this is the default case and if there is a reason why generated, custom navigators should not be called navigators.

---

Steps to reproduce

npx ignite-cli generate navigator test

### PR DESCRIPTION
**Pull request by simon-lund**
## Please verify the following:

- [ ] `yarn ci:test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR

Append "Navigator" to name of navigator component in navigator template. resolves #1854

**Note**: I tried to run tests (used both  `yarn test` and `yarn ci:test`) right after cloning the repo but it got stuck at *test/ignite-new.test.ts*, so this PR is not tested :cry:

### CODE REVIEW

File: boilerplate/ignite/templates/navigator/NAME-navigator.tsx.ejs [hunk 1]
```diff
@@ -9,7 +9,7 @@ export type <%= props.pascalCaseName %>ParamList = {
 }
 
 const Stack = createStackNavigator<<%= props.pascalCaseName %>ParamList>()
```
Comment (jamonholmgren) [inline boilerplate/ignite/templates/navigator/NAME-navigator.tsx.ejs:11]: Let's change the ParamList to *NavigatorParamList too.

```suggestion
const Stack = createStackNavigator<<%= props.pascalCaseName %>NavigatorParamList>()
```

Comment (jamonholmgren) [reply boilerplate/ignite/templates/navigator/NAME-navigator.tsx.ejs:11]: (And on line 7 as well...)

**Review by jamonholmgren — CHANGES_REQUESTED** [review]
Just a couple small changes!

File: boilerplate/ignite/templates/navigator/NAME-navigator.tsx.ejs (hunk 1 above)
Comment (simon-lund) [reply boilerplate/ignite/templates/navigator/NAME-navigator.tsx.ejs:11]: good point

**Review by jamonholmgren — APPROVED** [review]

**Pull request merged**